Fixes: Inconsistency of Light mode and Dark mode in screenshots

AjeshRPai opened the pull request:
Fixes #17133 

## Changes 

### Updates the logic to make all the screenshots to be in light mode. 

The logic of making the screen dark mode before taking a screenshot was changed to the light mode in `WPScreenshotTest.java` and `JPScreenshotTest.java`. 


## Testing Instructions

### To test:

Test 1 

- Run `WPScreenshotTest.java`
- Verify that all the screenshots are in light mode 


Test 2
- Run `JPScreenshotTest.java`. 
- Verify that all the screenshots are in light mode  


## Regression Notes
1. Potential unintended areas of impact
Screenshots are not in correct mode

2. What I did to test those areas of impact (or what existing automated tests I relied on)
Manual

3. What automated tests I added (or what prevented me from doing so)
N/A

PR submission checklist:

- [x] I have completed the Regression Notes.
- [x] I have considered adding accessibility improvements for my changes.
- [x] I have considered if this change warrants user-facing release notes and have added them to `RELEASE-NOTES.txt` if necessary.

AjeshRPai committed:
Fixes: Inconsistency of Light mode and Dark mode in screenshots

Updates the logic to make all the screenshots to be in light mode.

AliSoftware commented on WordPress/src/androidTest/java/org/wordpress/android/ui/screenshots/WPScreenshotTest.java:
```diff
@@ -277,8 +277,8 @@ private boolean editPostActivityIsNoLongerLoadingImages() {
 
-    private void setNightModeAndWait(boolean isNightMode) {
-        setNightMode(isNightMode);
+    private void setLightModeAndWait() {
```
I wonder if it's still useful to call `setLightModeAndWait()` before every single call to `takeScreenshot` if we're now only taking light mode screenshots for every screen and never dark modes?

Like, we could instead just switch to light mode at the very beginning of the test, by calling this method only once at the start, and not having to call it again before every individual `takeScreenshot` calls.

The purpose of this method back when it took a boolean was to be able to navigate to a screen, switch to night mode to take one screenshot, then switch to light mode to take another screenshot of the same screen, then continue navigating to the next screen, switch, screenshot, switch, screenshot, continue, etc. If we don't constantly switch mode anymore we could then avoid the `idleFor(5000)` that would happen before each screenshot, and speed up the tests there.

That is all assuming that the `idleFor(5000)` here didn't serve a double purpose of not only waiting for the mode to change from light to dark in case it took some time, but also to wait for the screen content (regardless of if light/night mode needed switching) finished loading? But if that;s case the case I'm not sure why we grouped those two lines together in a single function if there're not _both_ related to mode switching 🙃

AjeshRPai replied:
@AliSoftware Yup I agree, what was I thinking 😅 . 

>Like, we could instead just switch to light mode at the very beginning of the test, by calling this method only once at the start, and not having to call it again before every individual take screenshot calls.

yup 💯 . I will make that change and push. 


>That is all assuming that the idleFor(5000) here didn't serve a double purpose of not only waiting for the mode to change from light to dark in case it took some time, but also to wait for the screen content (regardless of if light/night mode needed switching)

IdleFor(5000) was only used for waiting for the mode to change. So it's safe to do the change you suggested.

AjeshRPai committed:
Moves: Switching to light mode logic to the parent function

AjeshRPai committed:
Moves: Light mode logic to parent function

Earlier the logic to set light mode was called from each function on screenshot. This has been moved to the parent function since all the screenshots needed are on Light mode.

zwarm reviewed (approved):
@AjeshRPai - I took it out for a spin and works as expected. I don't see any switching to dark mode. 👍  

If @AliSoftware does not want another review, then feel free to merge.
